src/requirements.h

```diff
@@ -190,6 +210,8 @@ struct requirement_data {
         const requirement_data disassembly_requirements() const;
 
     private:
+        std::string id_;
```

Comment:
Sometimes you're handed the `obj` rather than `id` for it. It is more convenient to be able to extract the id from it directly rather than having to manually wrap it in `string_id`.